### PR TITLE
[audit] timestamp.move

### PR DESCRIPTION
**Pull request by lightmark**
### Description
nits

### Test Plan

- - -
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/4619)

### CODE REVIEW

**Review by chloeqjz — APPROVED** [review]

File: aptos-move/framework/aptos-framework/sources/timestamp.move [hunk 1]
```diff
@@ -45,8 +45,8 @@ module aptos_framework::timestamp {
         } else {
             // Normal block. Time must advance
             assert!(now < timestamp, error::invalid_argument(EINVALID_TIMESTAMP));
+            global_timer.microseconds = timestamp;
         };
-        global_timer.microseconds = timestamp;
```
Comment (movekevin) [inline aptos-move/framework/aptos-framework/sources/timestamp.move:49]: This is unnecessary but is technically a no-op if (proposer == @vm_reserved) so more of a cleanup? :)

**Review by movekevin — APPROVED** [review]

File: aptos-move/framework/aptos-framework/sources/timestamp.move (hunk 1 above)
Comment (lightmark) [reply aptos-move/framework/aptos-framework/sources/timestamp.move:49]: save gas? lol.

Comment (movekevin) [reply aptos-move/framework/aptos-framework/sources/timestamp.move:49]: I'd not worried about gas for this one line. It's trivial.

Comment (lightmark) [reply aptos-move/framework/aptos-framework/sources/timestamp.move:49]: I am kidding tho

**Pull request merged**